backend/mediaprovider/model.go

```diff
@@ -179,4 +179,5 @@ type SearchResult struct {
 
 	// Unset for ContentTypes Artist, Playlist, and Genre
 	ArtistName string
+	Query      string
```

Comment:
Not sure about adding Query to the SearchResult model, since that isn't really part of the search result *per se*. Instead, maybe the searchdialog can expose the query through another function call (SearchDialog.QueryText()?)